modules/auxiliary/gather/microweber_lfi.rb

```diff
@@ -0,0 +1,179 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Microweber v1.2.10 Local File Inclusion (Authenticated)',
+        'Description' => %q{
+          Microweber v1.2.10 has a backup functionality. Upload and download endpoints can be combined to read any file from the filesystem.
+          Upload function may delete the local file if the web service user has access.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Talha Karakumru <talhakarakumru[at]gmail.com>'
+        ],
+        'References' => [
+          ['URL', 'https://huntr.dev/bounties/09218d3f-1f6a-48ae-981c-85e86ad5ed8b/']
+        ],
+        'Notes' => {
+          'SideEffects' => [ 'ARTIFACTS_ON_DISK', 'IOC_IN_LOGS' ],
+          'Reliability' => [ 'REPEATABLE_SESSION' ],
+          'Stability' => [ 'OS_RESOURCE_LOSS' ]
```

Comment:
Utilise constants, not strings.
```suggestion
          'SideEffects' => [ ARTIFACTS_ON_DISK, IOC_IN_LOGS ],
          'Reliability' => [ REPEATABLE_SESSION ],
          'Stability' => [ OS_RESOURCE_LOSS ]
```

Reply:
Fixed.